CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ if(POLICY CMP0051)
   cmake_policy(SET CMP0051 NEW)
 endif()
 
+if(POLICY CMP0054)
```

Comment:
Yes, it will be better for Cygwin.
I think other platforms don't have any effect to `CMP0054 NEW`, but it is not proven.
So, I'm preparing a PR in which the policy setting is guarded with Cygwin condition.